samples/42.scaleout/src/main/resources/log4j2.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Is this file correct? what is its usage?

Reply:
You are totally right, thanks for pointing this

Reply:
The file is correct, I'm not quite sure what's the goal right now, this is part of all the samples